setup.md

```diff
@@ -45,32 +45,37 @@ A video format of these setup instructions is available here:
 - [Docker Compose](https://docs.docker.com/compose/install/)
 - [jq](https://stedolan.github.io/jq/download/)
 
+## Requirements
+
+- Completion of the [Vertification of the GUAC images and
+  binaries]({{ site.baseurl }}{%link verification.md %})
+
 ## Step 1: Download GUAC
 
 1. Download the GUAC CLI `guacone` binary for your machine's OS and architecture
    from the
-   [latest GUAC release](https://github.com/guacsec/guac/releases/latest). For
-   example Linux x86_64 is
+   [latest GUAC release](https://github.com/guacsec/guac/releases/latest) if you
+   have not already done so. For example Linux x86_64 is
    [`guacone-linux-amd64'](https://github.com/guacsec/guac/releases/latest/download/guacone-linux-amd64).
 
-1. Rename the binary to `guacone`, mark it executable if necessary, and add it
+2. Rename the binary to `guacone`, mark it executable if necessary, and add it
    to your shell's path.
 
-1. Download the compose files from the
+3. Download the compose files from the
```

Comment:
Does your editor do this?

Reply:
yeah, not sure what caused the numbering to be weird...

Reply:
ah that I did not know. Let me change it back